packages/kbn-test/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@kbn/test",
+  "main": "./target/index.js",
+  "version": "1.0.0",
+  "license": "Apache-2.0",
+  "private": true,
+  "scripts": {
+    "build": "babel src --out-dir target",
+    "kbn:bootstrap": "yarn build"
+  },
+  "devDependencies": {
+    "@kbn/babel-preset": "link:../kbn-babel-preset",
+    "@kbn/dev-utils": "link:../kbn-dev-utils",
+    "babel-cli": "^6.26.0",
+    "getopts": "2.0.0",
+    "glob": "5.0.13",
```

Comment:
My reasoning was to use the same versions that are currently being used, without upgrading dependencies.

Reply:
Why does it matter what versions are currently used in Kibana if we introduce isolated (once we get rid of FTR dependency), test-only and brand new package with its own dependencies and have a chance to not rely on something that's outdated (both dependencies themselves and the way we define them in `package.json` since we commit `yarn.lock`)?

Just to be clear, it's not a big deal at all and we can discuss it after this PR is merged, I'm just trying to find common ground on this matter and not to raise this question again :)